gitlab-pipeline/stage/test.yml

```diff
@@ -597,25 +597,25 @@ test_full_integration_enterprise:
 
     - INTEGRATION_TEST_SUITE=$(integration/extra/release_tool.py --select-test-suite || echo "all")
     - if [ "$INTEGRATION_TEST_SUITE" = "$TEST_SUITE" ] || [ "$INTEGRATION_TEST_SUITE" = "all" ]; then
-    -   # Post job status
+        # Post job status
     -   ${CI_PROJECT_DIR}/scripts/github_pull_request_status pending "Gitlab ${CI_JOB_NAME} started" "${CI_JOB_URL}" "${CI_JOB_NAME}/${INTEGRATION_REV}"
 
     -   echo Running integration tests suite $INTEGRATION_TEST_SUITE
-    -   # only do automatic test suite selection if the user wasn't specific
-    -   # run.sh will pick up the SPECIFIC_INTEGRATION_TEST var
+        # only do automatic test suite selection if the user wasn't specific
+        # run.sh will pick up the SPECIFIC_INTEGRATION_TEST var
     -   if [ -z "$SPECIFIC_INTEGRATION_TEST" ]; then
     -     case $TEST_SUITE in
-    -       "enterprise")
+    - '     "enterprise")'
     -         export SPECIFIC_INTEGRATION_TEST="Enterprise";;
-    -       "open")
+    - '      "open")'
```

Comment:
Why does this need to be quoted?

Reply:
For some reason, the ")" matches the opening of the subshell in first line "(". TBH I am not sure if this fix is needed on top of the others, (difficult to verify again, due to not being able to lint any more), but this fix was the first I had to fix to be able to see the other error (the comments).

I'll merge and maybe follow-up later on.

Reply:
:+1: